test/src/test/resources/fixtures/hidden/can-query-non-hidden-fields.yml

```diff
@@ -0,0 +1,71 @@
+name: can query non hidden fields
```

Comment:
Unrelated to this diff, but can we add another test for `@hidden` fields not at the top level?

Only thing I'd be worried about testing there is that the top level field routing is somehow being the one handling the hidden fields, then nested fields are never checked.

e.g. we should have a test like

```
type Query {
  hello: World
}

type World {
  area51: Area @hidden
}
```

where the query is

```
{
  hello {
    area51
  }
}
```